docs/src/appendix/sncast/call.md

```diff
@@ -31,6 +31,12 @@ Optional.
 Inputs to the function, represented by a list of space-delimited values, e.g. `0x1 2 0x3`.
 Calldata arguments may be either 0x hex or decimal felts.
 
+## `--arguments`
+Optional.
+
+Function arguments provided as a comma-separated string of Cairo expressions. This provides a more human-readable way to pass arguments compared to raw calldata.
```

Comment:
I'd just limit it to that, we want appendix to be simple explanation of possible flags, not motivation behind them.

```suggestion
Function arguments provided as a comma-separated string of Cairo expressions. 
```